.github/workflows/caller.yml

```diff
@@ -26,13 +26,13 @@ jobs:
       operatorsdk_version: v1.18.0
       push_image: true
       release_version: v0.10.0
-      target_platforms: linux/amd64,linux/arm/v7,linux/arm64/v8
+      target_platforms: linux/amd64,linux/arm/v7,linux/arm64/v8, windows/amd64
```

Comment:
For windows build, just adding the windows/amd64 to the target platforms won't work because the list of platforms emulated by QEMU doesn't includes windows. Windows target has to be build separately and then added to the image manifest.